test/domain.middleware.test.ts

```diff
@@ -0,0 +1,55 @@
+import 'mocha';
+import { Request, Response, NextFunction } from 'express';
+import { expect } from 'chai';
```

Comment:
Order imports by length like a bharyang.

```js
import { expect } from 'chai';
import { Request, Response, NextFunction } from 'express';
```

Reply:
You mean order imports accordingly with the package name

```js
import { expect } from 'chai';
import { NextFunction, Request, Response } from 'express';
import 'mocha'; // <==== Is this OK
import { createRequest, createResponse } from 'node-mocks-http';
import * as globalStore from '../src';
import AsyncStoreAdapter from '../src/AsyncStoreAdapter';
```

Reply:
It should be like this https://github.com/leapfrogtechnology/async-store/blob/master/src/index.ts#L1-L10

First the external imports (packages, built-in modules) then after a blank line the other internal imports.